Description: store phonebook in /var/lib/fsogsmd/pb
 Use /var/lib/fsogsmd/pb to store the phonebook. Upstream's
 default location may be flushed during the boot. The new
 location complies with the FHS.
Author: Rico Rommel <rico@bierrommel.de>
Index: fso-gsmd/src/lib/phonebook.vala
===================================================================
--- fso-gsmd.orig/src/lib/phonebook.vala	2012-05-25 18:19:49.782093002 +0200
+++ fso-gsmd/src/lib/phonebook.vala	2012-05-25 18:26:27.818093001 +0200
@@ -21,7 +21,7 @@
 
 namespace FsoGsm
 {
-    public const string PB_STORAGE_DEFAULT_STORAGE_DIR = "/tmp/fsogsmd/pb";
+    public const string PB_STORAGE_DEFAULT_STORAGE_DIR = "/var/lib/fsogsmd/pb";
     public const int PB_STORAGE_DIRECTORY_PERMISSIONS = (int)Posix.S_IRUSR|Posix.S_IWUSR|Posix.S_IXUSR|Posix.S_IRGRP|Posix.S_IXGRP|Posix.S_IROTH|Posix.S_IXOTH;
 } /* namespace FsoGsm */
 
